fuel-vm/src/tests/validation.rs

```diff
@@ -35,3 +52,118 @@ fn transaction_can_be_executed_after_maturity() {
         .execute_tx(tx);
     assert!(result.is_ok());
 }
+
+/// Malleable fields should not affect validity of the block
+#[test]
+fn malleable_fields_do_not_affect_validity() {
+    let rng = &mut StdRng::seed_from_u64(8586);
+
+    let params = ConsensusParameters::default();
+
+    let tx_size_ptr =
+        32 + (params.tx_params.max_inputs as usize * (AssetId::LEN + WORD_SIZE));
+    let tx_start_ptr = tx_size_ptr + 8;
```

Comment:
These might be good candidates for some kind of GTF or GM helper in the future 💡